content/en/docs/community/related.md

```diff
@@ -140,9 +140,6 @@ Tools layered on top of Helm.
 Platforms, distributions, and services that include Helm support.
 
 - [Kubernetic](https://kubernetic.com/) - Kubernetes Desktop Client
-- [Cabin](https://www.skippbox.com/cabin/) - Mobile App for Managing Kubernetes
-- [Qstack](https://qstack.com)
-- [Fabric8](https://fabric8.io) - Integrated development platform for Kubernetes
```

Comment:
In the Fabric8 docs they refer to their Helm integration. Why is it being dropped as a related project? Should this one instead be kept? See https://fabric8.io/docs/index.html

Reply:
The entire fabric8 project is archived and the docs can go away at a moment's notice. IMHO links to archived projects are worse than no links.

Reply:
Interesting: https://github.com/fabric8io/fabric8